### ISSUE TITLE
mermaid in slide mode cuts text

### ISSUE THREAD

**Issue by pedro-nonfree** [hedgedoc/react-client#983]
**Description**

mermaid in slide mode cuts text

my temporary solution was to switch to graphviz (there I have more size control)

**To Reproduce**

(mermaid codeblock in hedgedoc document)

```
graph TD
Cluster--> Service1
Cluster--> Service2
Cluster--> Service3
Cluster--> Service4
```

**Expected behavior**

in preview looks fine:

![image](https://user-images.githubusercontent.com/39959198/105562988-ccb7d800-5d1c-11eb-86a6-a840383e9aec.png)

https://demo.hedgedoc.org/IY7EXiIEQDOCB-G3DWMkWw#

but in slide mode (presentation) text is cutted

![image](https://user-images.githubusercontent.com/39959198/105563001-db9e8a80-5d1c-11eb-9c64-a718817e95b9.png)

https://demo.hedgedoc.org/p/7zChCxg40#/

**Comment by davidmehren** [comment]
I moved this issue to our new frontend, so we can make sure to fix this issue in the 2.0 release.

**Comment by ErikMichelson** [comment]
This has been fixed in the frontend of HedgeDoc 2.0.

![image](https://user-images.githubusercontent.com/52606896/176764565-9b24f102-e70a-4757-872b-970b445fa474.png)
